### ISSUE TITLE
Git asks me to report a bug

### ISSUE THREAD

**Issue by edreamleo** [git-for-windows/git#3009]
- [x] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?

```
$ git --version --build-options

git version 2.16.2.windows.1
cpu: x86_64
built from commit: e1848984d1004040ec5199e749b5f282ddf4bb09
sizeof-long: 4
```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

```
$ cmd.exe /c ver

Microsoft Windows [Version 10.0.19041.746]
```

 - What options did you set as part of the installation? Or did you choose the
   defaults?

```
# One of the following:

> type "C:\Program Files\Git\etc\install-options.txt"

The system cannot find the path specified.

> type "C:\Program Files (x86)\Git\etc\install-options.txt"

The system cannot find the path specified.

> type "%USERPROFILE%\AppData\Local\Programs\Git\etc\install-options.txt"

The system cannot find the path specified.
```

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

Not that I know of.

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

cmd

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
gc
```

This is bound to c:\scripts\gc.bat, containing only `git commit %*` This command has never failed before.

 - What did you expect to occur after running these commands?

That scite.exe would open.

 - What actually happened instead?

```
c:\leo.repo\leo-editor>c:\apps\Git\bin\git.exe commit
hint: Waiting for your editor to close the file... warning: failed to restrict file handles (2)

handle #0: 0000000000000054 (type 2, handle info (1) 0
handle #1: 0000000000000120 (type 2, handle info (1) 1
handle #2: 0000000000000124 (type 2, handle info (1) 1

This is a bug; please report it at
https://github.com/git-for-windows/git/issues/new

To suppress this warning, please set the environment variable

        SUPPRESS_HANDLE_INHERITANCE_WARNING=1

error: cannot spawn c:/bin/scite.bat: No such file or directory
error: unable to start editor 'c:/bin/scite.bat'
Please supply the message using either -m or -F option.
```

 - If the problem was occurring with a specific repository, can you provide the
   URL to that repository to help us with testing?

https://github.com/leo-editor/leo-editor.git

In more detail:

```
c:\leo.repo\leo-editor>gc -m "Improve g.run_unit_tests_externally"

c:\leo.repo\leo-editor>c:\apps\Git\bin\git.exe commit -m "Improve g.run_unit_tests_externally"
[devel 8e0233e7b] Improve g.run_unit_tests_externally
 3 files changed, 33 insertions(+), 7 deletions(-)

c:\leo.repo\leo-editor>push

c:\leo.repo\leo-editor>git push

c:\leo.repo\leo-editor>c:\apps\Git\bin\git.exe push
Counting objects: 8, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (8/8), done.
Writing objects: 100% (8/8), 1.07 KiB | 547.00 KiB/s, done.
Total 8 (delta 7), reused 0 (delta 0)
remote: Resolving deltas: 100% (7/7), completed with 7 local objects.
To https://github.com/leo-editor/leo-editor.git
   b3f5876cb..8e0233e7b  devel -> devel
```

**Comment by rimrul** [comment]
This should be fixed in 2.17.0 and newer. Feel free to reopen if you still experience this issue in a recent version.

**Comment by edreamleo** [comment]
@rimrul Thanks for the quick reply. Good to know it has been fixed.
